doc/binning_basics.rst

```diff
@@ -0,0 +1,102 @@
+Using the binning module
+========================
+
+Stile contains some simple functions to bin your data.  Two of them--:class:`stile.BinStep
+<stile.binning.BinStep>` and :class:`stile.BinList <stile.binning.BinList>`--have simple,
```

Comment:
Could use some context for how/what this is used for -- i.e., which systematics tests might require binning.